[Caffe2] Add RMSNormOp

xiaomengy opened the pull request:
Summary: Add RMSNormOp in Caffe2

Test Plan: buck test mode/dev-nosan //caffe2/caffe2/python/operator_test:rms_norm_op_test

Differential Revision: D23546424

houseroad reviewed (approved):
py3 linter issue is unrelated.

xiaomengy committed:
[Caffe2] Add RMSNormOp (#44338)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/44338

Add RMSNormOp in Caffe2

Test Plan: buck test mode/dev-nosan //caffe2/caffe2/python/operator_test:rms_norm_op_test

Reviewed By: houseroad

Differential Revision: D23546424

fbshipit-source-id: 31a9b0efedc9a59fc96c31333f41568a777960cc
